### PR TITLE
Implementing a descending order view of the message list

### PR DESCRIPTION
**Pull request by tsikerdekis**
The request implements the new feature options. I wasn't able to test it extensively due to the send email error #19 . I added a few more files on .gitignore since I use Codelite on Linux.

### CODE REVIEW

**Comment by mayurdw** [comment]
Looks good to me, we can't test it of course

**Comment by dweremeichik** [comment]
@tsikerdekis I gave it a try, seems nice! The verbiage is a bit confusing however. I would like to see the description for the option be made more clear. Perhaps change message in "Display message list in descending order" to "Conversation", "Thread" or  "responses". I was driving myself crazy because I thought it was supposed to invert the main messages window.

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,9 @@ scripts/venv
 
 # OAuth client secret for talking to Google Sheets
 client_secret.json
+
+# CodeLite Workspace Files
+.tern-port
+.tern-project
+nylas-mail.workspace*
+.codelite/*
```
Comment (dweremeichik) [inline .gitignore]: If we are going to add CodeLite to the gitignore we should probably do it in a different PR.

Comment (tsikerdekis) [inline .gitignore]: Good idea, I'll delete the changes for codelite from this pull request and create a new one

**Comment by tsikerdekis** [comment]
I pushed everything on my master which messes up changing virtually anything. I am closing this pull request without committing and will be submitting a new one with the modified changes.
